src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java

```diff
@@ -23,6 +23,7 @@ public class Game {
     private final GameMode gameMode;
     private final GameType gameType;
     private final Map<String, PlayerDecorator> decoratedPlayers;
+    private int turnIndex;
```

Comment:
Maybe we should encapsulate turnIndex and turnNumber into a class where you set the total number of players in the constructor.
With two getters for turnIndex and turnNumber.
And one method for nextTurn.

I am just a bit scared that we miss to increase one field somewhere and then it doesn't work anymore.. :|

Reply:
Done, let me know your opinion.

Reply:
Perfect ty! I think that's saver 👍